### PR TITLE
add docstrings

### LINKED ISSUE #49: Write docstrings for each of the classes

### PR DESCRIPTION
**Pull request by nathansit**
Just annotating one class. If it looks good will add the rest
Closes #49

### CODE REVIEW

File: eslint.config.mjs
Comment (gideontong) [inline eslint.config.mjs:1]: Let's not modify files that aren't part of the original issue unless there's a super good reason

**Review by gideontong — CHANGES_REQUESTED** [review]
it's good enough

File: eslint.config.mjs
Comment (nathansit) [reply eslint.config.mjs:1]: I only changed it because `npx eslint .` was complaining

Comment (nathansit) [reply eslint.config.mjs:1]: But ok I just reverted it

File: src/classes/Book.ts [hunk 1]
```diff
@@ -1,3 +1,12 @@
+/**
+ * Represents a Book in the Bible
```
Comment (gideontong) [inline src/classes/Book.ts]: Lowercase book

File: src/classes/Book.ts [hunk 2]
```diff
@@ -1,3 +1,12 @@
...
+ * @property {string} name name of the Book
+ * @property {Map<string, string>} otherNames alternative names for Book
+ * @property {number} chapterCount number of chapters in Book
+ * @property {Book} prev the previous Book of the Bible
+ * @property {Book} next the next Book of the Bible
+ * @property {object} metadata data about Book
```
Comment (gideontong) [inline src/classes/Book.ts]: More specific description

File: src/classes/Chapter.ts [hunk 3]
```diff
@@ -1,3 +1,7 @@
+/**
+ * A subunit of a Book
```
Comment (gideontong) [inline src/classes/Chapter.ts]: That's not really a true description

File: src/classes/Chapter.ts [hunk 4]
```diff
@@ -1,3 +1,7 @@
...
+ * @property {object} metadata data about Chapter
```
Comment (gideontong) [inline src/classes/Chapter.ts]: More descriptive please

File: src/classes/Reference.ts [hunk 5]
```diff
@@ -1,3 +1,7 @@
+/**
+ * Pointer to a related Passage from the Bible
```
Comment (gideontong) [inline src/classes/Reference.ts]: This isn't true

File: src/classes/Verse.ts [hunk 6]
```diff
@@ -1,3 +1,7 @@
+/**
+ * A subunit of Chapter, which is a subunit of Book
```
Comment (gideontong) [inline src/classes/Verse.ts]: Try to use more descriptive language and not "subunit"

File: src/classes/Version.ts [hunk 7]
```diff
@@ -1,3 +1,8 @@
+/**
+ * The Bible has many different Versions across different translations.
+ * This class stores information to keep track of which is currently being used
```
Comment (gideontong) [inline src/classes/Version.ts]: That isn't true

**Review by gideontong — CHANGES_REQUESTED** [review]

File: src/classes/Passage.ts [hunk 8]
```diff
@@ -1,3 +1,7 @@
+/**
+ * A set of one or more consecutive verses from the Bible
```
Comment (gideontong) [inline src/classes/Passage.ts:2]: What exactly are we annotating here

File: src/classes/Chapter.ts [hunk 9]
```diff
@@ -1,3 +1,8 @@
+/**
+ * A book of the Bible consists of one or more chapters. These manmade delineations divide
```
Comment (gideontong) [inline src/classes/Chapter.ts:2]: Be more clear you are describing a chapter

File: src/classes/Verse.ts [hunk 10]
```diff
@@ -1,3 +1,9 @@
+/**
+ * A book is comprised of chapters. These chapters can be further divided into verses.
```
Comment (gideontong) [inline src/classes/Verse.ts:2]: Be more clear you're describing a verse

File: src/classes/Version.ts [hunk 11]
```diff
@@ -1,3 +1,7 @@
+/**
+ * The Bible has many different Versions across different translations.
```
Comment (gideontong) [inline src/classes/Version.ts:2]: Do not capitalize versions unless it is exactly the class name?

**Review by gideontong — CHANGES_REQUESTED** [review]

**Comment by gideontong** [comment]
Sorry I didn't get to this earlier, next time you can request review so I get a notification as I never got a notification.

**Comment by nathansit** [comment]
push

**Comment by nathansit** [comment]
make unstable

**Pull request closed without merging**